### PR TITLE
Added hover properties and media queries

### PR DESCRIPTION
**Pull request by MaxGranger97**

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -13,6 +13,10 @@ body {
 	height: 80px;
 }
 
+.logo-div {
+
+}
+
```
Comment (bradleypurdue) [inline portfolio.css]: There's no reason for this to be here.

File: portfolio.html [hunk 2]
```diff
@@ -68,7 +70,9 @@ <h3>Project 4</h3>
 		
 		<section class="sm-section">
 			<a id="contact-me"></a>
-			<img class="contact-image" src="file:///Users/Max/Downloads/1485969_796556610358000_706157516_o%20(2).jpg" alt="contact me image">
+			<div>
+				<img class="contact-image" src="file:///Users/Max/Downloads/1485969_796556610358000_706157516_o%20(2).jpg" alt="contact me image">
```
Comment (bradleypurdue) [inline portfolio.html]: Might be a good idea to move this image into the product folder as it won't be avaliable for anyone else.

**Review by bradleypurdue — CHANGES_REQUESTED** [review]

File: portfolio.html [hunk 3]
```diff
@@ -13,18 +13,20 @@
...
+			<div class="logo-div">
+				<img class="logo" src="https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/089641db-8d3d-40f6-92e8-1744e0272fed/db18sam-17538ca1-c20f-43c2-aeec-e74871345a14.png/v1/fill/w_1024,h_820,strp/clean_wolf_logo_by_akther_brothers__no_watermark__by_shoaibakther_db18sam-fullview.png?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9ODIwIiwicGF0aCI6IlwvZlwvMDg5NjQxZGItOGQzZC00MGY2LTkyZTgtMTc0NGUwMjcyZmVkXC9kYjE4c2FtLTE3NTM4Y2ExLWMyMGYtNDNjMi1hZWVjLWU3NDg3MTM0NWExNC5wbmciLCJ3aWR0aCI6Ijw9MTAyNCJ9XV0sImF1ZCI6WyJ1cm46c2VydmljZTppbWFnZS5vcGVyYXRpb25zIl19.bQPKV239OIZBAPo4MBe1UlZ6l4St10WXzg03HogOS98" alt="Max G Logo">
+			</div>
 			<div class="container anchors">
```
Comment (danicatt) [inline portfolio.html:19]: Instead of a single space, you could use an hyphen i.e. <div class="container-anchors"

**Review by danicatt — COMMENTED** [review]
Looks good to me.

**Pull request closed without merging**